authors/settings/local.py

```diff
@@ -0,0 +1,4 @@
+from authors.settings.base import *
```

Comment:
Why do we have an empty file, it does not seem to do anything apart from import everything from base.